app-arch/unar/unar-1.10.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs bash-completion-r1
+
+DESCRIPTION="unpacker for various archiving formats, e.g. rar v3"
+HOMEPAGE="https://unarchiver.c3.cx/"
+SRC_URI="
+	https://github.com/MacPaw/XADMaster/archive/v${PV}/XADMaster-${PV}.tar.gz
+	https://github.com/MacPaw/universal-detector/archive/1.1/universal-detector-1.1.tar.gz
+"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	media-sound/wavpack
+	app-arch/bzip2:=
+	dev-libs/icu:=
+	gnustep-base/gnustep-base
+	sys-libs/zlib:=
+"
+DEPEND="${RDEPEND}
+	app-arch/unzip
```

Comment:
I suspect this is a BDEPEND. However I see unar.zip isn't being used in this ebuild, so maybe this is not needed during build-time at all? Maybe it belongs to RDEPEND for .zip support?

Reply:
Seems to me it is unused. I can not find any references to it in my unpacked workdir.